style.css

```diff
@@ -30,69 +19,82 @@ body > div {
 h1 {
     font-size: 24px;
     font-weight: 950;
-    color: #F9FAF8;
-    margin-right: auto;
+    color: #f9faf8;
+}
+
+.nav {
+    display: flex;
+    gap: 24px;
 }
 
 .heroMainText {
     font-size: 48px;
     font-weight: bolder;
-    color: #F9FAF8;
+    color: #f9faf8;
 }
 
 .top {
     display: flex;
     padding-top: 24px;
-    justify-content: end;
+    justify-content: space-between;
     gap: 32px;
+    align-items: center;
 }
 
 h2 {
     font-size: 48px;
-    color: #F9FAF8;
+    color: #f9faf8;
     font-weight: 1000;
 }
 
-.topHalfSplit {
+.top-half-split {
     display: flex;
-    gap: 25px;
+    flex-wrap: wrap;
+    gap: 40px;
 }
 
 .splitChild {
-    width: 50%;
-    height: 220px;
+    flex: 1;
```

Comment:
`flex: 1` means "take up the available space", and for two flex children they'll take up half the available space. So the same end result, but the difference is when there is only enough room for one child.

`width: 50%`:

![CleanShot 2024-05-16 at 11 28 28](https://github.com/VamanaPed/LandingPage/assets/199204/e8631a8a-1841-47ac-aea3-a5cb992cccb3)

`flex: 1`:

![CleanShot 2024-05-16 at 11 28 14](https://github.com/VamanaPed/LandingPage/assets/199204/3ac7ad2a-96ff-44ad-acbe-483d979b40b5)